ui/components/app/modals/account-details-modal/account-details-modal.component.js

```diff
@@ -94,22 +102,25 @@ export default class AccountDetailsModal extends Component {
             data: address,
           }}
         />
+        {process.env.MULTICHAIN ? null : (
+          <>
+            <div className="account-details-modal__divider" />
 
-        <div className="account-details-modal__divider" />
-
-        <Button
-          type="secondary"
-          className="account-details-modal__button"
-          onClick={
-            blockExplorerLinkText.firstPart === 'addBlockExplorer'
-              ? routeToAddBlockExplorerUrl
-              : openBlockExplorer
-          }
-        >
-          {this.context.t(blockExplorerLinkText.firstPart, [
-            blockExplorerLinkText.secondPart,
-          ])}
-        </Button>
+            <Button
+              type="secondary"
```

Comment:
Note: We recently merged an update to the `Button` component that changes the `type` prop to `variant`
```suggestion
              variant={BUTTON_VARIANT.SECONDARY}
```